src/main/java/fi/otavanopisto/kuntaapi/server/integrations/ptv/AbstractPtvProvider.java

```diff
@@ -507,7 +558,18 @@ private OrganizationId translateOrganizationId(String ptvOrganizationId) {
     
     return organizationKuntaApiId;
   }
-
+  
+  private ServiceId translateServiceId(String ptvServiceId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "translateServiceId" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)